integration_tests/load_tests.feature

```diff
@@ -16,19 +16,26 @@ Feature: Load Tests
   Scenario: DefinedLoadStrategy trades energy based on csv provided profile
     Given we have a scenario named strategy_tests/user_profile_load_csv
     And d3a is installed
-    When we run the d3a simulation with strategy_tests.user_profile_load_csv [24, 15, 15]
-    Then the DefinedLoadStrategy follows the Load profile provided as csv
+    When we run the d3a simulation with strategy_tests.user_profile_load_csv [None, 24, 15, 15]
+    Then the DefinedLoadStrategy follows the single day Load profile provided as csv
+    And load only accepted offers lower than final_buying_rate
+
+  Scenario: DefinedLoadStrategy trades energy based on a multiday csv profile
+    Given we have a scenario named strategy_tests/user_profile_load_csv_multiday
+    And d3a is installed
+    When we run the d3a simulation with strategy_tests.user_profile_load_csv_multiday [2019-01-01, 24, 15, 15]
```

Comment:
Since, this integrationtest targets multiday csv profile. It should probably be tested by running simulation for more then one day

Reply:
You are totally right! Will change that.